.rubocop.yml

```diff
@@ -19,3 +19,6 @@ AllCops:
     - vendor/**/*
   NewCops: enable
   TargetRubyVersion: 2.7
+
+Style/RedundantCurrentDirectoryInPath:
+  Enabled: false
```

Comment:
Please move it to https://github.com/ixti/sidekiq-throttled/blob/main/rubocop/style.yml#L47